src/templates/_header.pug

```diff
@@ -19,4 +19,4 @@ header.header
         svg
           use(xlink:href="#icon-twitter")
     .button__block
-      a.button.button--login(href="/details/index.pug") ENTRAR
+      a.button.button__orange.button__orange--login(href="/details/index.pug") ENTRAR
```

Comment:
@PinhoOn remember we don't use `.button__orange.button__orange--login` but `.button--orange.button--login` instead. Double underscore only if you have something as a child like:

```
.button
   .button__icon
```